utils/src/server/rpc_module.rs

```diff
@@ -74,21 +77,82 @@ struct SubscriptionKey {
 
 /// Callback wrapper that can be either sync or async.
 #[derive(Clone)]
-pub enum MethodCallback {
+enum MethodKind {
 	/// Synchronous method handler.
 	Sync(SyncMethod),
 	/// Asynchronous method handler.
 	Async(AsyncMethod<'static>),
 }
 
+/// Information about resources the method uses during its execution.
```

Comment:
```suggestion
/// Information about resources the method uses during its execution. Lazily initialized when the the server starts.
```

Reply:
Sans `Lazily`, it's doing opposite of lazy initialization atm.